README.md

```diff
@@ -100,6 +100,7 @@ The following are quickstarts. These are written using libvirt, but are generic
 * DHCP install [quickstart](docs/quickstart.md)
 * Static IPs install [quickstart](docs/quickstart-static.md)
 * DHCP install on Power [quickstart](docs/quickstart-ppc64le.md)
+* DHCP install on PowerVM [quickstart](docs/quickstart-powervm.md)
```

Comment:
It might be better to rename "DHCP install on Power" to "DHCP install on KVM/Power" to avoid any confusion.

Reply:
sure